.zenodo.json

```diff
@@ -0,0 +1,232 @@
+{
+  "title": "NetCDF Climate and Forecast (CF) Metadata Conventions",
+  "description": "This document describes the CF conventions for climate and forecast metadata designed to promote the processing and sharing of files created with the netCDF Application Programmer Interface. The conventions define metadata that provide a definitive description of what the data in each variable represents, and of the spatial and temporal properties of the data. This enables users of data from different sources to decide which quantities are comparable, and facilitates building applications with powerful extraction, regridding, and display capabilities. The CF conventions generalize and extend the COARDS conventions. The extensions include metadata that provides a precise definition of each variable via specification of a standard name, describes the vertical locations corresponding to dimensionless vertical coordinate values, and provides the spatial coordinates of non-rectilinear gridded data. Since climate and forecast data are often not simply representative of points in space/time, other extensions provide for the description of coordinate intervals, multidimensional cells and climatological time coordinates, and indicate how a data value is representative of an interval or cell. This standard also relaxes the COARDS constraints on dimension order and specifies methods for reducing the size of datasets.",
+  "license": "CC0-1.0",
+  "creators": [
+    {
+      "name": "Brian Eaton",
+      "affiliation": "NCAR"
+    },
+    {
+      "name": "Jonathan Gregory",
+      "affiliation": "University of Reading and UK Met Office Hadley Centre",
+      "orcid": "0000-0003-1296-8644"
+    },
+    {
+      "name": "Bob Drach",
+      "affiliation": "PCMDI, LLNL"
+    },
+    {
+      "name": "Karl Taylor",
+      "affiliation": "PCMDI, LLNL",
+      "orcid": "0000-0002-6491-2135"
+    },
+    {
+      "name": "Steve Hankin",
+      "affiliation": "PMEL, NOAA"
+    },
+    {
+      "name": "Jon Blower",
+      "affiliation": "University of Reading"
+    },
+    {
+      "name": "John Caron",
+      "affiliation": "Unidata"
+    },
+    {
+      "name": "Rich Signell",
+      "affiliation": "USGS"
+    },
+    {
+      "name": "Phil Bentley",
+      "affiliation": "UK Met Office"
+    },
+    {
+      "name": "Greg Rappa",
+      "affiliation": "MIT"
+    },
+    {
+      "name": "Heinke Höck",
+      "affiliation": "DKRZ"
+    },
+    {
+      "name": "Alison Pamment",
+      "affiliation": "CEDA"
```

Comment:
"orcid": "0000-0001-5040-4626"

Reply:
Hi Gui @castelao

Thanks for working on this. I've added a comment with my ORCID - please could you include it in the pull request?

Best wishes,
Alison

Reply:
Could you double-check if it's correct, please?